bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RegionAwareEnsemblePlacementPolicy.java

```diff
@@ -173,6 +174,10 @@ public RegionAwareEnsemblePlacementPolicy initialize(ClientConfiguration conf,
         super.initialize(conf, optionalDnsResolver, timer, featureProvider, statsLogger, bookieAddressResolver)
                 .withDefaultRack(NetworkTopology.DEFAULT_REGION_AND_RACK);
         myRegion = getLocalRegion(localNode);
+        String localRegion = conf.getString(REPP_LOCAL_REGION, null);
```

Comment:
simple code 
`String localRegion = conf.getString(REPP_LOCAL_REGION);`
instead of
`String localRegion = conf.getString(REPP_LOCAL_REGION, null);`

<img width="703" alt="image" src="https://user-images.githubusercontent.com/42990025/181058870-f6bc2165-ee33-494a-8a89-2afd88dcb60a.png">

Reply:
I'm not sure whether  `throwExceptionOnMissing` could be set true in some condition.